css/color-scheme.css

```diff
@@ -455,6 +429,22 @@ body {
 button + button {
     margin-inline-start: 1rem;
 }
+
+/* code */
+
+.code-base {
+    --max-spc-width: 600px;
+    margin-inline-start: 0;
+}
+
+.code-base>figcaption {
+    font-size: 0.55rem;
+    padding-block-start: 1rem;
+    padding-inline-start: 1.2rem;
+    padding-inline-end: 1.2rem;
```

Comment:
could have used padding-inline

Reply:
Some browsers do not support short hands or sometimes bug out, so I preferred this way